images/docker-compose.yml.in

```diff
@@ -161,7 +161,7 @@ services:
             - eve
 
     sshd:
-        image: linuxkit/sshd:v0.5
+        image: linuxkit/sshd:666b4a1a323140aa1f332826164afba506abf597
```

Comment:
why are you changing this -- but even if you have a good reason (unless it is somehow related to containerd) I'd rather discuss it in a separate PR